### ISSUE TITLE
Tray notifications do not respect Show display name in title flag

### ISSUE THREAD

**Issue by odhil** [runelite/runelite#17580]
### Type

Incorrect behavior

### Operating System

None

### Bug description

Settings have an option to remove username from the window title "usernameInTitle". However, it has no effect on the title in tray notifications defined [here](https://github.com/runelite/runelite/blob/51aab9cd6f814ceadcf4bf614f0347270abf070e/runelite-client/src/main/java/net/runelite/client/Notifier.java#L218).

### RuneLite version

LuneLite launcher version: 2.6.13

**Comment by Nightfirecat** [comment]
I don't think this behavior is necessarily a bug. What circumstances do you want the ign to not be displayed for a given client's notifications?

**Comment by odhil** [comment]
You're probably right it is intended behavior. Regardless, such a setting already exists and I think it should also apply to the notifications. The format in the notification title is same as in the window title. If the user wishes to not show the username in the window title, the user most likely does not need to distinguish from which account the notification comes from either.

To answer your question, I want the ign not be displayed in the notification if the setting "Show display name in title" is not set.

**Comment by Nightfirecat** [comment]
I guess I should be more direct--*why* do you not want notifications to show the ign for the client firing the notification? Is this a privacy concern, or is there some other reason? I see the reasoning behind linking the setting to affect both pieces of functionality, however I'm not convinced that is necessarily a strong enough reason to change this behavior.

**Comment by odhil** [comment]
The main reason is to declutter the notifications. Having less information makes it easier to grasp the core message at a glance. My use case, a bit of a niche, is forwarding the notifications to a watch with a small screen making the notification body to but cut off.
